libursa/Database.cpp

```diff
@@ -99,19 +93,22 @@ void Database::save() {
     db_file.open(db_base / tmp_db_name, std::ofstream::binary);
 
     json db_json;
-    db_json["config"] = {{"max_mem_size", max_memory_size}};
+    db_json["config"] = std::map<std::string, std::string>();
 
     std::vector<std::string> dataset_names;
     for (const auto *ds : working_datasets) {
         dataset_names.push_back(ds->get_name());
     }
     db_json["datasets"] = dataset_names;
 
-    json iterators_json;
+    json iterators_json = std::map<std::string, std::string>();
     for (const auto &it : iterators) {
         iterators_json[it.first] = it.second.get_name().get_filename();
     }
     db_json["iterators"] = iterators_json;
+    // Explicit version instead of ursadb_version, because this is is a
+    // *file format version*, not necessarily equal to the db version.
+    db_json["version"] = "1.3.2";
```

Comment:
maybe as a constant and together with other similar constants?

Reply:
:+1: